src/Microsoft.AspNet.Mvc.Core/Rendering/HtmlHelperInputExtensions.cs

```diff
@@ -101,7 +103,7 @@ public static HtmlString CheckBox([NotNull] this IHtmlHelper htmlHelper, string
         /// "hidden" with value "false".
         /// </summary>
         /// <param name="htmlHelper">The <see cref="IHtmlHelper"/> instance this method extends.</param>
-        /// <param name="name">Expression name, relative to the current model.</param>
+        /// <param name="expression">Expression name, relative to the current model.</param>
```

Comment:
why?  I prefer not to make a change here because this PR is about parameter naming, not about the docs.  I made the changes to avoid `expression expression` and similar oddities but don't think much more is in scope.